### PR TITLE
pangram.py

### PR DESCRIPTION
**Pull request by amymhaddad**

### CODE REVIEW

File: pangram.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+def is_pangram(sentence):
+    """Is a sentence a pangram?"""
+
+    alpha_list_lowercase = string.ascii_lowercase
+    sentence_without_spaces = ''
+
+    if sentence == '':
+        return False
+
+    for letter in sentence:
```
Comment (paulghaddad) [inline pangram.py]: Nice job here!

File: pangram.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        lowercase_letter = letter.lower()
+        if lowercase_letter != ' ':
+            sentence_without_spaces += lowercase_letter
+
+    for lowercase_letter in alpha_list_lowercase:
+        if lowercase_letter not in sentence_without_spaces:
+            return False
```
Comment (paulghaddad) [inline pangram.py]: Need a line break to separate the `return True` as it is not part of the `for` loop.

File: pangram.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    if sentence == '':
+        return False
+
+    for letter in sentence:
+        lowercase_letter = letter.lower()
+        if lowercase_letter != ' ':
+            sentence_without_spaces += lowercase_letter
+
+    for lowercase_letter in alpha_list_lowercase:
```
Comment (paulghaddad) [inline pangram.py]: Really nice job iterating through all the lowercase letters! And immediately returning `False` and exiting the function if it's not in the sentence. You have your answer right then and there.

File: pangram.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+"""Test to see if a sentence is a pangram?"""
+import string
+
+def is_pangram(sentence):
+    """Is a sentence a pangram?"""
+
+    alpha_list_lowercase = string.ascii_lowercase
```
Comment (paulghaddad) [inline pangram.py]: Consider making this a simpler name: `lowercase_letters`.

**Comment by amymhaddad** [comment]
@paulghaddad I updated this exercise:
-Changed the variable name to the list of letters. However I have a question about this: shouldn't `lowercase_letters` be a constant? Pylint didn't say to make it a constant, but I think it should be. Thoughts?

-Added space at the end of the file for `Return True`

**Pull request merged**